scripts/pathfinders_validation.py

```diff
@@ -0,0 +1,559 @@
+        "Unlocking and enabling industrial and commercial development",
+        "Strengthening the visitor and local service economy",
+        "Improving the quality of life of residents",
+        "Employment & Education",
+    ]
+    RISK_CATEGORIES = [
+        "Strategy risks",
+        "Programme or project delivery risks",
+        "People risks",
+        "Operational process or control risks",
+        "Resilience risks",
+        "Governance risks",
+        "System or IT infrastructure risks",
+        "Security risks",
+        "Financial risks",
+        "Commercial or procurement risks",
+        "Lega, regulatory or compliance risks",
+        "Arm’s length body risks",
+        "Local government risks",
+        "Political sensitivity",
+        "Reputational Risk",
+        "Resource & Expertise",
+        "Slippage",
+        "Planning Permission / other consents",
+        "Procurement / contracting",
+        "Delivery Partner",
+        "Financial flexibility",
+        "Subsidy Control",
+        "Other",
+    ]
+    RISK_SCORES = ["1 - very low", "2 - low", "3 - medium", "4 - high", "5 - very high"]
+    FORECAST_ACTUAL = ["Forecast", "Actual"]
+    SPEND_TYPE = [
+        "How much of your forecast is contractually committed?",
+        "Freedom and flexibilities spend",
+        "Total DLUHC spend (incl F&F)",
+        "Secured Match Funding Spend",
+        "Unsecured Match Funding ",
+        "Total Match ",
+    ]
+    REPORTING_PERIOD = [
+        "Q1 Apr - Jun 23/24",
+        "Q2 Jul - Sep 23/24",
+        "Q3 Oct - Dec 23/24",
+        "Q4 Jan - Mar 23/24",
+        "Q1 Apr - Jun 24/25",
+        "Q2 Jul - Sep 24/25",
+        "Q3 Oct - Dec 24/25",
+        "Q4 Jan - Mar 24/25",
+        "Q1 Apr - Jun 25/26",
+        "Q2 Jul - Sep 25/26",
+        "Q3 Oct - Dec 25/26",
+        "Q4 Jan - Mar 25/26",
+        "Apr 26 onwards",
+    ]
+
+
+class PFErrors:
+    is_float = (
+        "You entered text instead of a number. Remove any units of measurement and only use numbers, for example, 9."
+    )
+    is_datetime = "You entered text instead of a date."
+    isin = (
+        "You’ve entered your own content, instead of selecting from the dropdown list provided. Select an "
+        "option from the dropdown list."
+    )
+    lt_100_words = "Please enter a maximum of 100 words."
+
+
+PF_CONFIG = {
+    "Financial Completion Date": {
+        "extract": {"id_tag": "PF-TABLE-ID-01", "worksheet_name": "Admin"},
+        "process": {
+            "ignored_non_header_rows": [0],
+        },
+        "validate": {
+            "columns": {
+                "Pathfinder Financial Completion Date": pa.Column(
+                    datetime, pa.Check.is_datetime(error=PFErrors.is_datetime)
+                ),
+            }
+        },
+    },
+    "Practical Completion Date": {
+        "extract": {"id_tag": "PF-TABLE-ID-02", "worksheet_name": "Admin"},
+        "process": {"ignored_non_header_rows": [0]},
```

Comment:
All formatting is done automatically using [black](https://black.readthedocs.io/en/stable/)

Reply:
This is true, but for black (at least with the current configuration) there is no difference between this:
`"process": {"ignored_non_header_rows": [0]}`

And this:
`"process": {
   "ignored_non_header_rows": [0],
}`

Both variations pass the black check, so it's up to us to enforce this consistency. It's only a minor thing but just adding this here so we're on the same wavelength

Reply:
ah we must have different configurations because when I run black it formats to the above

Reply:
although that is odd - as we should be using the same pre commit config..

Reply:
What I mean is that if you run black on `scripts/pathfinders_validation.py` with the current config formatting, it will pass, but then if you make the change to have `"process": {"ignored_non_header_rows": [0]}` on one line for "Financial Completion Date" and then run black again, it will also pass. So our current black config isn't opinionated on this, so it's up to us to be consistent (within reason)

Reply:
```
"process": {
     "ignored_non_header_rows": [0]
},
```
Pretty sure I get what you're saying but this doesn't pass black for me and automatically formats it to 
```
"process": {"ignored_non_header_rows": [0]},
```

Reply:
oh, I see, if you leave a comma after the last item then formats it differently